src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,30 @@
+import ThumbsDown from "./ThumbsDown";
+import ThumbsUp from "./ThumbsUp";
+import { Link } from "react-router-dom";
+
+const ReviewCard = ({ reviews }) => {
+    return (
+    <ul>
+    {reviews.map((review) => (
+    <li key={review.review_id} className="reviewCard">
+        <Link to={`/review/${review.review_id}/comments`}>
+        <img src={review.review_img_url} className="review_image" alt="Avatar"></img>
+        </Link>
+    <h2>{review.title}</h2>
+    <p>Review ID: {review.review_id}</p>
+    <p>Category: {review.category}</p>
+    <p>Designer: {review.designer}</p>
+    <p>Owner: {review.owner}</p>
+    <p>Review: {review.review_body}</p>
```

Comment:
This is a matter of preference, but do you want to display the body in the reviews list? Would it be better to display the body when an individual review is selected?